packages/components/forms/src/base-checkbox/BaseCheckbox.styles.ts

```diff
@@ -2,31 +2,49 @@ import { css } from 'emotion';
 import tokens from '@contentful/f36-tokens';
 import type { BaseCheckboxInternalProps } from './types';
 
+const sizeToStyle = (size) => {
+  if (size === 'small') {
+    return {
+      height: tokens.spacingM,
+      width: tokens.spacingXl,
+    };
+  }
+
+  return {
+    height: '20px',
+    width: '40px',
+  };
+};
+
 const getStyles = ({
   isDisabled,
   type,
-}: Pick<BaseCheckboxInternalProps, 'isDisabled' | 'type'>) => ({
+  size,
+}: Pick<BaseCheckboxInternalProps, 'isDisabled' | 'type' | 'size'>) => ({
```

Comment:
uhhhh smart!

I will use this in my branches too hahahah I mean the Pick<> strategy